src/action/base/create_or_insert_into_file.rs

```diff
@@ -79,11 +79,12 @@ impl CreateOrInsertIntoFile {
                 let discovered_mode = discovered_mode & 0o777;
 
                 if discovered_mode != mode {
-                    return Err(ActionError::PathModeMismatch(
-                        this.path.clone(),
+                    tracing::debug!(
```

Comment:
Should this maybe be a warning? Or maybe not, since if the permissions prevent us from doing stuff, we'll likely error out later with something like "cannot write to file" or something so nvm

Reply:
I am not sure. We are running as `root` so if we can't access something that's bad news anyways for us ya.